packages/cli/test/unit/index.test.ts

```diff
@@ -1,11 +1,38 @@
 import { describe, expect, it, vi, beforeEach, afterEach } from 'vitest';
+import './test/mocks/matchers';
 
 import { Output } from '../../src/util/output';
 import { TelemetryEventStore } from '../../src/util/telemetry';
 import { TelemetryBaseClient } from '../../src/util/telemetry/base';
 
 describe('main', () => {
   describe('telemetry', () => {
+    describe('version', () => {
+      it('tracks version', () => {
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏test the negative case as well